yoga/Yoga.cpp

```diff
@@ -39,24 +39,24 @@ static int YGDefaultLog(const YGConfigRef config,
 #endif
 
 static YGConfig gYGConfigDefaults = {
-    .experimentalFeatures =
+    /* .experimentalFeatures = */
```

Comment:
There's still lots of designated initializer syntax to be removed, but I at least added the YGStyle and YGLayout components.